sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/codegen/CodeGenerator.scala

```diff
@@ -916,6 +916,10 @@ object CodeGenerator extends Logging {
     cache.get(code)
   }
 
+  def refresh(code: CodeAndComment) : Unit = {
+    cache.refresh(code)
+  }
+
```

Comment:
Why refresh and not invalidate? The refresh looks a dicey to me (async loading with previous value still valid etc).